CHANGELOG.md

```diff
@@ -2,16 +2,27 @@
 All notable changes to [Solang](https://github.com/hyperledger/solang/)
 will be documented here.
 
-## Unreleased
+## v0.2.1 Rio
+
+### Added
+- The Anchor IDL data structure is now generated for every Solana contract, although the actual IDL json file is not yet saved.
+[LucasSte](https://github.com/LucasSte)
+- Many pesky bugs have been fixed by [seanyoung](https://github.com/seanyoung), [LucasSte](https://github.com/LucasSte)
```

Comment:
I'm not sure about pesky. Also, I don't think this belongs under Added.  Maybe it should go under Fixed.